Dockerfile

```diff
@@ -1,8 +1,10 @@
 FROM eywalker/pydev
 
-MAINTAINER Fabian Sinz <sinz@bcm.edu>
```

Comment:
Sorry I overwrote it with a file I already had - didn't mean it to make a name claiming thing :(